ElfUtils/ElfFile.cpp

```diff
@@ -200,10 +217,25 @@ std::string ElfFileImpl<ElfT>::GetFilePath() const {
   return file_path_;
 }
 
+template <typename ElfT>
+ErrorMessageOr<LineInfo> ElfUtils::ElfFileImpl<ElfT>::GetLineInfo(uint64_t address) {
+  auto line_info_or_error = symbolizer_.symbolizeCode(
+      *object_file_, {address, llvm::object::SectionedAddress::UndefSection});
+  if (!line_info_or_error) {
```

Comment:
I actually do not know when it returns a error.. maybe we should add detection of `<invalid>` and instead of returning `{"<invalid>", 0}` generate a error message.

Reply:
I am in favor of generating an ErrorMessage for this case

Reply:
I briefly looked into the llvm source code and it seems that `symbolizeCode` only returns an error when there is something wrong with the object file.

Reply:
done.